### DISCUSSION TITLE
Fief doesn't let a user ('Default' Tenant) with Admin rights into Admin Panel after some time

### DISCUSSION THREAD

**Discussion by dzmitrysyrtsou** [fief-dev/fief#439] [Q&A] [answered]
Hi everyone! 😊

We're experiencing an issue with Fief in a system that has two Tenants (Default and a Custom one). Several users are registered under the Default Tenant, and these users have Administrator permissions assigned to them.

After some time (a few days for one user and a couple of weeks for others), these users are no longer able to access the Fief Admin Panel (e.g., /admin/users). Instead, they encounter the following error:

![image](https://github.com/user-attachments/assets/02720062-2af3-4d90-8501-76f759e241e1)

Has anyone experienced something similar or have insights into what might be causing this? Could it be due to a misconfiguration on our end?

Thanks in advance for your help! 🙏

**Comment by fief-bailiff** [comment]
Hail, @dzmitrysyrtsou 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!

**Comment by frankie567** [comment]
Hi!

This error means the server is not able to find the [Client](https://docs.fief.dev/configure/clients/) to authenticate users. A Fief server, when starting, always initialize with a default client and tenant; following the values set in [environment variables](https://docs.fief.dev/self-hosting/environment-variables/#fief-ception).

Make sure you didn't delete the default client from the admin dashboard. If you did, you can stop your server, run the command `fief migrate` and restart your server. This should make sure the default client is created.

**Reply by StefanFVogel** [reply]
Well ... 

Dzmitry's user was working, for several Weeks.
While mine and the initial user (from the environment variables) had exact this Problem.
We tried several times to create new admin users which have exact the same Problem.

how to start the `fief migrate` on a containerized installation?

Cheers
   Stefan

**Reply by frankie567** [reply]
Did you check that the client with ID defined by `FIEF_CLIENT_ID` environment variable is still there?

To run a command in your Docker container, you can do something like this, assuming the Fief server container is named `fief-server`

```sh
docker compose exec fief-server fief migrate
```

**Reply by StefanFVogel** [reply, marked as answer]
Thank you Frankie..

FIEF_CLIENT_ID has been overwritten in the secrets file with the FIEF_CLIENT_ID of our application!
After changing the FIEF_CLIENT_ID of our application the service is back!

Cheers
   Stefan
